test/functional/autocmd/termclose_spec.lua

```diff
@@ -22,6 +20,7 @@ describe('TermClose event', function()
   end)
 
   it('triggers when long-running terminal job gets stopped', function()
+    if helpers.pending_win32(pending) then return end
     nvim('set_option', 'shell', 'sh')
```

Comment:
I don't know why `sh` is required here so I kept this disabled in Windows.

Reply:
It was probably just copy/pasted from the other tests below.

Reply:
@jamessan 
Not my code AFAIR?!
Maybe @justinmk amended something, if your comment is based on git-blame?!

Reply:
If it passes, ok to remove.